packages/adapters/wagmi/src/client.ts

```diff
@@ -701,6 +700,7 @@ export class WagmiAdapter implements ChainAdapter {
 
             this.appKit?.setPreferredAccountType(preferredAccountType, chainNamespace)
             this.appKit?.setCaipAddress(caipAddress, chainNamespace)
+            this.appKit?.setStatus(status, chainNamespace)
```

Comment:
@KannuSingh I guess we might need to do same for disconnect status, could you do that as well?

Reply:
okay, I will check and do the same.